torch/nested/_internal/ops.py

```diff
@@ -304,6 +316,14 @@ def jagged_binary_pointwise(func, *args, **kwargs):
             return NestedTensor(
                 func(a._values, b._values, *args[2:], **kwargs), **extract_kwargs(a)
             )
+
+        # ex: (B, j1, D) + (B, j2, D)
+        if has_different_symint_size(a, b._size):
```

Comment:
I think `raggedness_matches` already checks this. 

Sorry for not being clear on the original issue. I think what I'm looking for is to add to the existing mismatch error message with extra information about exactly what the user should do in these situations. 
e.g., someone needs  to workout what's the best workaround the recommend to the user here. It's tricky because not sure we something super easy to just plug and play.

Going to remove "actionable" for now from the original issue.